apps/publisher/src/components/publisher-video-view/video-control-bar/index.tsx

```diff
@@ -0,0 +1,124 @@
+import { HStack } from '@chakra-ui/react';
+import React from 'react';
+
+import {
+  IconCameraOff,
+  IconCameraOn,
+  IconMicrophoneOff,
+  IconMicrophoneOn,
+  IconPause,
+  IconPlay,
+  IconSoundOff,
+  IconSoundOn,
+} from '@millicast-react/dolbyio-icons';
+import IconButton from '@millicast-react/icon-button';
+import StatisticsPopover from '@millicast-react/statistics-popover';
+
+import { VideoControlBarProps } from './types';
+import SettingsPopover from './settings-popover';
+import LiveIndicator from '@millicast-react/live-indicator';
+import { StreamTypes } from '@millicast-react/use-multi-media-streams';
+
+const VideoControlBar = ({
+  activeAudio,
+  activePlayback,
+  activeVideo,
+  canTogglePlayback,
+  hasAudioTrack,
+  hasVideoTrack,
+  isConnecting = false,
+  isStreaming = false,
+  onStartLive: handleStartLive,
```

Comment:
start/stop live is belong to app-level, and shouldn't be a property of the video-view, I suggest video-control-bar provides an interface for the app to put any UI elements in it.

Reply:
Let's solve it in #291